### PR TITLE
Try some more debugging in case of snapper fails

### PR DESCRIPTION
**Pull request by okurz**
Could help with for example bsc#1032831.

Verification run: http://lord.arch/tests/6308

### CODE REVIEW

File: lib/x11regressiontest.pm [hunk 1]
```diff
@@ -484,6 +484,10 @@ sub start_gnome_settings {
     send_key "super";
     wait_still_screen;
     type_string "settings";
+    # we need to give the gnome dash a second to settle down before trying to
+    # click an entry which is not responsive yet and does not show the preview
+    # icon yet
+    wait_still_screen(3);
```
Comment (asmorodskyi) [inline lib/x11regressiontest.pm]: 'magic number' - please extract to variable

File: tests/x11/yast2_snapper.pm [hunk 2]
```diff
@@ -135,13 +136,25 @@ sub run() {
 
 sub post_fail_hook {
     my ($self) = @_;
+    # snapper actions can put the system under quite some load so we want to
+    # give some more time, e.g. for login in the consoles
+    my $previous_timeout_scale = get_var('TIMEOUT_SCALE', 1) * 10;
```
Comment (asmorodskyi) [inline tests/x11/yast2_snapper.pm]: in backend we have function scale_timeout .... not sure , but probably will be more correct to call it here ? Anyway I don't like some magic formula which you need to support in one single place

File: tests/x11/yast2_snapper.pm [hunk 3]
```diff
@@ -135,13 +136,25 @@ sub run() {
...
+    select_console('root-console', await_console => 0);
+    sleep 10;
```
Comment (asmorodskyi) [inline tests/x11/yast2_snapper.pm]: 'magic number' please extract to variable

File: tests/x11/yast2_snapper.pm [hunk 4]
```diff
@@ -135,13 +136,25 @@ sub run() {
...
     # Upload y2log for analysis if yast2 snapper fails
     assert_script_run "save_y2logs /tmp/y2logs.tar.bz2";
     upload_logs "/tmp/y2logs.tar.bz2";
     save_screenshot;
+    diag('check if at least snapper low-level commands still work');
+    script_run('snapper ls');
+    diag('Collect a backtrace from potentially running snapperd, e.g. for bsc#1032831');
+    script_run('pidof snapperd && gdb --batch -q -ex "thread apply all bt" -ex q /usr/sbin/snapperd $(pidof snapperd) |& tee /tmp/snapperd_bt_all.log');
+    upload_logs '/tmp/snapperd_bt_all.log';
+    set_var('TIMEOUT_SCALE', $previous_timeout_scale);
```
Comment (asmorodskyi) [inline tests/x11/yast2_snapper.pm]: or I miss something or you just extract value to a variable to put it back without using it ?

Comment (okurz) [inline tests/x11/yast2_snapper.pm]: the value is used implicitly in the testapi functions, e.g. assert_screen in between

**Comment by okurz** [comment]
LGTM by asmorodskyi

**Pull request merged**